cert_manager/acme.py

```diff
@@ -175,6 +175,28 @@ def add_domains(self, acme_id, domains):
 
         return result.json()
 
+    def get_domains (self, acme_id, **kwargs):
+        """List ACME account’s domains.
+
+        :param int acme_id: The ID of the acme account to list domains
+        """
+        self.__acc_domains = []
+        result = self.__find_domains(acme_id)
+        for dom in result:
+            self.__acc_domains.append(dom)
+        return self.__acc_domains
+
+    @paginate
+    def __find_domains(self, acme_id, **kwargs):
+
+        params = {
+          self._find_params_to_api[param]: kwargs.get(param)
+          for param in self._find_params_to_api  # pylint:disable=consider-using-dict-items
+        }
+        url = self._url(f"{acme_id}", "domain")
+        result = self._client.get(url, params=params)
+        return result.json()
+
```

Comment:
This should all probably be one function named `get_domains` that uses the `@paginate` wrapper. Additionally, we probably shouldn't cache the results in `self.__acc_domains` since this data would change if a different ACME ID was provided, correct? This should probably just accumulate a list of domains in a local variable and then return at the end.